shallowblue.py

```diff
@@ -255,6 +263,26 @@ def init_malmo(agent_host):
     return agent_host
 
 
+# inspired by fellow team MineFarm-Farmer: https://yongfeitan.github.io/MineFarm-Farmer/status.html
+def get_entities(entities, agentX, agentZ):
+    resources = dict()
+    for ele in entities:
+        if ele['name'] in RESOURCES:
+            X = ele['x']
+            Y = ele['y']
+            Z = ele['z']
+
+            # if agent is center of 5x5 square, any entity can only be at most 2.5 units away
+            if _dist(agentX, agentZ, X, Z) <= 2.5:
```

Comment:
not sure if dist should be < 5? i tried drawing it out and i think it should be 2.5